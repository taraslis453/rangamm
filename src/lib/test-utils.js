import React from 'react'
import {render} from '@testing-library/react'
import {theme} from '../theme'
import {ThemeProvider} from 'styled-components'
import {BrowserRouter} from 'react-router-dom'
import {store} from '../store'
import {Provider} from 'react-redux'
const AllTheProviders = ({children}) => {
  return (
    <div id='root'>
      <BrowserRouter>
        <Provider store={store}>
          <ThemeProvider theme={theme}>{children}</ThemeProvider>
        </Provider>
      </BrowserRouter>
    </div>
  )
}

const customRender = (ui, options) => render(ui, {wrapper: AllTheProviders, ...options})
export * from '@testing-library/react'

export {customRender as render}
