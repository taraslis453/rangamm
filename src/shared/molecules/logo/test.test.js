import React from 'react'
import {render} from 'lib/test-utils'
import {Logo} from './index'
describe('<Logo/>', () => {
  it('renders properly', () => {
    const {container} = render(<Logo />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
