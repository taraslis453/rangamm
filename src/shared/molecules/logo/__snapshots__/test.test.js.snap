// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo/> renders properly 1`] = `
<div
  id="root"
>
  <div
    class="sc-dlfnbm ebCEzK"
  >
    <a
      href="/"
    >
      <img
        alt=""
        height="40px"
        src="react-icon.png"
      />
    </a>
    <div
      class="sc-hKgILt gsUhK"
      display="none,block"
    >
      <span
        class="sc-gKsewC eifmwT"
      >
        Free photos for everyone
      </span>
    </div>
  </div>
</div>
`;
